src/network/send.ts

```diff
@@ -186,6 +197,20 @@ export async function sendMessage({
     logging.Network
   )
   const realSatsIndex = contactIds.findIndex((cid) => cid === realSatsContactId)
+  if (realSatsContactId && realSatsIndex < 0) {
+    await sleep(1000)
+    const originalMessage = (await models.Message.findOne({
+      where: { tenant, uuid: msg.message.uuid },
+    })) as MessageRecord
+    return await reversePayment({
+      tenant,
+      originalMessage,
+      msgToBeSent: msg,
+      error: 'user is no longer in tribe',
+      amount,
+      sender,
+    })
```

Comment:
This bit of code looks pretty similar to the one below, I wonder if we could make this a function instead of repeating twice

Reply:
Sure...

Lemme fix that...